android/src/test/java/com/revenuecat/purchases/hybridcommon/CommonKtTests.kt

```diff
@@ -65,4 +71,120 @@ internal class CommonKtTests {
             setProxyURLString("this is not a url")
         }
     }
+
+    @RepeatedTest(5)
```

Comment:
@aboedo found this "repeatedtest" thing so that this test will run 5 times. thought it was a good way to check that we don't just get lucky with the random bool, since i t's a 50/50 chance. other option would be to add this verification to each of the other unit tests and take advantage of them running anyway, but that's only 3x. let me know what you think!